### ISSUE TITLE
Key `PrintScreen` does not being captured

### ISSUE THREAD

**Issue by snowman** [mpv-player/mpv#11889]
### Important Information

Provide following Information:
- mpv version `0.35.1`
- Windows Version `Windows 10 Version 21H2 (OS Build 19044.1826)`
- Source of the mpv binary `scoop`

### Reproduction steps
```sh
$ mpv --no-config --input-keylist | grep -i print
PRINT
$ mpv --no-config --input-test --player-operation-mode=pseudo-gui
```

### Expected behavior
When press key `SPACE`, it shows:

>  Key SPACE is bound to:
>  1. 'cycle pause' in <builtin>:1 (default)

When Press key `PrintScreen`, nothing shows up.
It should shows text `Key PRINT is bound to: ...` when press key `PrintScreen`

### Actual behavior
Nothing shows when press key `PrintScreen`

**Comment by Traneptora** [comment]
I can reproduce this on Linux X11 as well.

**Comment by Dudemanguy** [comment]
Are you sure your window manager isn't just taking the PrintScreen key? It works for me when I remove it from the bindings. No clue about windows of course.

**Comment by Traneptora** [comment]
Mine is grabbing it, not sure if there's a way around that.

**Comment by llyyr** [comment]
This works for me on both i3 and sway if I unbind any compositor keybinds that capture the PRINT key. I don't know if there's anything you can do to unbind printscreen on Windows though.

**Comment by Dudemanguy** [comment]
Yeah I don't think this is a linux issue. Your window manager and/or compositor should allow mpv to receive the key (if they don't, go bug them). Possibly, there's an actual windows issue of course.

**Comment by butterw** [comment]
On Win10, I can confirm that nothing happens when you press PrintScrn. It copies a screenshot to clipboard as you expect instead.

**Comment by Traneptora** [comment]
There is a windows API for disabling screen capture on the compositor for specific application views: https://learn.microsoft.com/en-us/samples/microsoft/windows-universal-samples/disablingscreencapture/

Do we want to implement it?
